docs/quick-start.md

```
@@ -51,9 +51,15 @@ When you run `agent-gauntlet`, it detects which entry points have changed files
 agent-gauntlet run
 ```
 
+- **Run gates from your agent and auto-fix detected issues**
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)